src/clarity-angular/checkboxes/checkbox.ts

```diff
@@ -82,14 +86,17 @@ export class Checkbox implements ControlValueAccessor {
     public change = new EventEmitter<boolean>(false);
 
     public toggle() {
-        this._indeterminate = false;
```

Comment:
If we set the private property here, how do we emit the `indeterminateChange` output?